### PR TITLE
Lusine: Tasks 221_ 230

### PR DESCRIPTION
**Pull request by hsp-lusine**
I skip tasks 222, 223, 225, 226, 227, 230.... because this tasks similar other tasks

### CODE REVIEW

File: src/main/java/am/hsp/apps/seua/base/TaskArrayBase.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import java.util.Random;
+
+/**
+ * Base class for all the array. Will contain general methods.
+ * 
+ * @author lusine
+ *
+ */
+public abstract class TaskArrayBase extends TaskBase {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/base/TaskArrayBase.java]: I wrote comments for this class in previous PR

File: src/main/java/am/hsp/apps/seua/task221_230/Task_221.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	 * 
+	 * @see main.java.am.hsp.apps.seua.base.TaskBase#solve()
+	 */
+	@Override
+	public void solve() {
+		int startRange = getUserInput("Please enter start of range");
+		int endRange = getUserInput("Please enter end of range");
+		int size = getUserInput("Please enter size of array");
+
+		int begin_A = getUserInput("Please enter start A of range");
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_221.java]: 1. Wrong variable name format

File: src/main/java/am/hsp/apps/seua/task221_230/Task_221.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		int end_B = getUserInput("Please enter start B of range");
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_221.java]: end of the range

File: src/main/java/am/hsp/apps/seua/task221_230/Task_221.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+	/**
+	 * Check number is in range
+	 * 
+	 * @param number
+	 * @param begin
+	 * @param end
+	 * @return
+	 */
+	private boolean isElementenvironment(int number, int begin, int end) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_221.java]: What???? What is the name of this method?????

File: src/main/java/am/hsp/apps/seua/task221_230/Task_224.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+	/**
+	 * Return the sum cubes of elements whose absolute value is less than the
+	 * specified number
+	 * 
+	 * @param array
+	 * @param number
+	 * @return
+	 */
+	private int getSumCupesOfComplyTheCondition(int[] array, int number) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_224.java]: CuBe

File: src/main/java/am/hsp/apps/seua/task221_230/Task_224.java [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+	}
+
+	/**
+	 * Check absolute value of number is less than the specified number
+	 * 
+	 * @param number
+	 * @param specifiedNumber
+	 * @return
+	 */
+	private boolean isLessNumber(int number, int specifiedNumber) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_224.java]: 1. method name is not indicating that the absolute value should be compared. isAbsLess
2. Second param name is bad. You need to somehow indicate that this parameter is for comparing your actual number with.
for example: numberToCompareWith

File: src/main/java/am/hsp/apps/seua/task221_230/Task_229.java [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+	}
+
+	/**
+	 * Check the difference between the value and the index a positive number
+	 * 
+	 * @param number
+	 * @param specifiedNumber
+	 * @return
+	 */
+	private boolean isPositiveNumber(int number, int index) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_229.java]: Bad method name. It just says is the number positive. The method has 2 parameters. So which one it should check is positive or note?
So method checks whether subtraction is positive or not...

File: src/main/java/am/hsp/apps/seua/task221_230/Task_229.java [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+	}
+
+	/**
+	 * Check the difference between the value and the index a positive number
+	 * 
+	 * @param number
+	 * @param specifiedNumber
+	 * @return
+	 */
+	private boolean isPositiveOrNot(int number, int index) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/task221_230/Task_229.java]: You didn't fix my comment.
Method name MUST indicate that it checks whether SUBTRACTION of its params is positive or not.

Comment (hsp-lusine) [reply src/main/java/am/hsp/apps/seua/task221_230/Task_229.java]: Can name method  "isNumberSubtractionIndexPositiveOrNot"

**Pull request merged**